cmd/cloudFoundryDeploy_test.go

```diff
@@ -1299,7 +1299,10 @@ func TestMtaExtensionCredentials(t *testing.T) {
 	extends: test
 	parameters
 		test-credentials1: "<%= testCred1 %>"
-		test-credentials2: "<%= testCred2 %>"`)
+		test-credentials2: "<%=testCred2%>"
```

Comment:
one content for all tests! i'd separate but i don't insist.

Reply:
done, test content is now defined inside each test. We repeat a little but, but might be easier to read and to understand.